vm/src/builtins.rs

```diff
@@ -312,19 +313,37 @@ fn builtin_id(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
 
 // builtin_input
 
-fn builtin_isinstance(obj: PyObjectRef, typ: PyClassRef, vm: &VirtualMachine) -> PyResult<bool> {
-    vm.isinstance(&obj, &typ)
+fn type_test(
```

Comment:
Could we think of a clearer function name, like `single_or_tuple_must_be_true`? Not required for the rest of the change, just the function name which is a little vague.